lib/components/DevCard.js

```diff
@@ -63,10 +63,18 @@ var DevCard = React.createClass({
   },
   render: function() {
     var name = this.props.name;
+    var namespace = this.props.namespace;
     var doc = this.props.doc;
     var body = this.props.body;
     var opts = this.props.options || {};
 
+    var href = [
```

Comment:
I'd quite like to extract this and the other URL making function into `routes.js`, that way the unit tests can round-trip